app/workers/notification_worker.rb

```diff
@@ -6,12 +6,13 @@ def perform(notification_params)
 
     tags_hash  = Hash(notification_params[:tags])
     links_hash = Hash(notification_params[:links])
-    document_type = notification_params[:document_type]
 
     query = SubscriberListQuery.new(
-      tags_hash,
-      links_hash,
-      document_type,
+      tags: tags_hash,
+      links: links_hash,
+      document_type: notification_params[:document_type],
+      email_document_supertype: notification_params[:email_document_supertype],
+      government_document_supertype: notification_params[:government_document_supertype],
```

Comment:
The notifications controller isn't passing the supertype params to the worker yet - is that intentional?